src/features/calendar/components/CalendarDayView/Day/Event.tsx

```diff
@@ -40,91 +43,96 @@ const Event = ({ event }: { event: ZetkinEvent }) => {
   }
 
   return (
-    <NextLink href={getEventUrl(event)} passHref>
-      <Link color="inherit" underline="none">
-        <Box
-          display="flex"
-          flexDirection="row"
-          justifyContent="space-between"
-          padding={2}
-          sx={{ backgroundColor: 'white', borderRadius: '3px' }}
-          width="100%"
-        >
-          <Box alignItems="center" display="flex" gap={2.2}>
-            {/* Status */}
+    <Box sx={{ backgroundColor: 'white', borderRadius: '3px' }}>
+      <Box alignItems="center" display="flex">
+        <Box sx={{ pb: 0.1, pl: 2, pr: 1 }}>
```

Comment:
For me it looked slightly uneven. So I added pb:0.1
![image](https://github.com/zetkin/app.zetkin.org/assets/77925373/048e7d38-4f81-4d56-b82b-eec7382d61b5)

Reply:
That's weird! It made no difference for me, but I guess it's a rounding issue then. I think you can leave it! But I'd still like the horizontal margins to be margins, because they aren't there to extend the checkbox, they're there to create gaps around the checkbox.